schema/types/suricata.schema

```diff
@@ -459,6 +459,46 @@ type suricata.rdp = suricata.component.common + record {
   }
 }
 
+type suricata.rfb = suricata.component.common + record {
+  rfb: record {
+    client_protocol_version: record {
+      major: string,
+      minor: string
```

Comment:
Why are the major and minor version fields strings?

Reply:
Because they are strings in the protocol and could theoretically be non-integers. See https://tools.ietf.org/html/rfc6143#section-7.1.1

Reply:
So the `x`s and `y`s in there can technically be any ASCII character. Do you mind adding a comment with these 2 links?

Reply:
Well, they would be JSON-escaped of course, But yes, they could be, Sure, I can add a comment.